stardog_fetcher.py

```diff
@@ -0,0 +1,37 @@
+import os
+import subprocess
+import time
+
+context_str = """kube1
+kube2
+kube3"""
+
+contexts = context_str.split('\n')
+
+for kubecontext in contexts:
+    print("working on {}".format(kubecontext))
+    os.mkdir("/tmp/{}".format(kubecontext))
+
+    filename = "/tmp/{}/pods.json".format(kubecontext)
+    print("writing to {}".format(filename))
+    subprocess.run(args=[
+        "bash", "-c",
+        "kubectl --context={} get pods -A -o json > {}".format(kubecontext, filename)
+        ])
+    time.sleep(2)
```

Comment:
what's the purpose of sleeping for two seconds here? Does subprocess.run not run until completion?

Reply:
I would not like to totally beat down kube